### DISCUSSION TITLE
SharePlugin sources.

### DISCUSSION THREAD

**Discussion by haschu** [ionic-team/capacitor#3624] [Q&A] [answered]
Hey, 

I just noticed, that the SharePlugin sources are somehow gone (at least I can't find them anymore in this repo)... will this plugin still be supported in the future?

Thanks!

**Comment by haschu** [comment, marked as answer]
Nevermind...

Plugin sources are here:
https://github.com/ionic-team/capacitor-plugins/tree/main/share

**Reply by imhoffd** [reply]
That repo is for Capacitor 3. Capacitor 2 plugins are still in the 2.x branch: https://github.com/ionic-team/capacitor/tree/2.x

More info: https://github.com/ionic-team/capacitor/blob/main/CONTRIBUTING.md
